config/routes.rb

```diff
@@ -23,7 +23,7 @@
     post "/orders/:status" => "orders#filter", as: "admin_filter_order"
     get "/orders/:status" => "orders#filter", as: "admin_order"
     patch "/categories/:id" => "categories#update"
-    resources :categories, only: [:create, :edit, :destroy, :new, :index]
```

Comment:
One, it's not a part of what we're told, so I removed it to clean up our routes a bit more, and two, it won't let us destroy a category unless it has nothing in it. I figured we can punt the issue.